next.config.js

```diff
@@ -14,3 +15,20 @@ module.exports = withBundleAnalyzer({
     ]
   },
 })
+
+const SentryWebpackPluginOptions = {
```

Comment:
The whole config has been automatically generated from `@sentry/wizard` for a Next.js project.

Reply:
We can assume that works then. 😄

Reply:
We are sure it's working since I forced an issue with that strange button I added right after creating the PR 😅